tools/test-case-generators/distro-arch-imagetype-map.json

```diff
@@ -64,6 +64,21 @@
             "qcow2-customize",
             "vhd",
             "vmdk"
+        ],
+        "aarch64": [
+            "ami",
+            "openstack",
+            "qcow2",
+            "rhel-edge-commit",
+            "tar"
+        ],
+        "ppc64le": [
+            "qcow2",
+            "tar"
+        ],
+        "s390x": [
+            "qcow2",
+            "tar"
         ]
     }
 }
```

Comment:
Missing newline at EOF. Otherwise looks good.

Reply:
Thanks for the review! Do you want me to add the newline as part of this PR or is this not a blocker from your point of view?

Reply:
OK, since I feel lucky and to get this PR in ideally today, I fixed the newline at EOF and pushed the change...